portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,34 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Andrea Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click <a href="/hello">here</a> to see content created by server code.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <div class="topNavigationBar">
+        <a class="active" href="./index.html">About me</a>
+        <a href="./gallery.html">gallery</a>
+        <a href="./projects.html">projects</a>
+      </div>
+      <div id="content">
+        <img src="/images/andrearamirez.jpg" border="0px">
+        <h1>Andrea Ramírez Portfolio 👩🏽‍💻</h1>
+        <p>
+            I am Computer Science student at Tecnológico de Monterrey from Venezuela 🇻🇪<br> 
+            Currently enrolled in the Bachelor of Computer Technologies Honors Program <br> 
+            🧚🏽‍♀️ In love with learning and working with others 💙 <br>
+            My greatest interests in tech are Mobile Development and Human Computer Interaction <br>                
+            In my free time I enjoy dancing and doing outdoor activities 🏞️ <br>
+            And while you are here you cand add me 😉
+            <!--Linkes of github + linkedin-->
+        </p>
+        <p>Click here for a random fun fact about me:</p>
+        <button onclick="addRandomGreeting()">Surprise me!</button>
```

Comment:
You might want to change the name of this function as it no longer shows a greeting but a random fact